datalad/crawler/nodes/misc.py

```diff
@@ -56,14 +57,19 @@ def get_values(self, *keys):
         return [[d[k] for k in keys] for d in self.data]
 
     def __call__(self, data):
-        # ??? for some reason didn't work when I made entire thing a list
         if self.keys:
-            raise NotImplementedError("Jason will do it")
+            for k in data.keys():
+                size = len(self.keys)
+                for i in range(0, size):
+                    key = self.keys[i]
+                    if key == k:
+                        self.data.append({key: data.get(key)})
```

Comment:
And then will come the challenge! Read about dict comprehensions in python, to see how you could do all needed here in 1 line ;-)